tests/test_methods.py

```diff
@@ -145,7 +145,7 @@ def checkMaskedImage(self, mi):
                     mi1 = mi.Factory(mi, True)
                     plane0 = getattr(mi0, getName)()
                     plane1 = getattr(mi1, getName)()
-                    plane1[2, 2] = errVal
+                    plane1[2, 2, afwImage.PARENT] = errVal
```

Comment:
see comment in `test_background.py`. There's no need to include the origin since `PARENT` is the default value.